src/extension/SettingsPanel.ts

```diff
@@ -4,12 +4,15 @@ import { getNonce } from './utils/getNonce'
 import { createAndOpenConfFile } from './utils/createAndOpenConfFile'
 import { log, Sources } from './utils/log'
 import { CommandFromSettingsWebview, EventFromSettingsWebview } from './types'
+import { settings } from 'cluster'
```

Comment:
What's this? Where do you use it?

Reply:
nowhere this is a mistake I'll remove it